llama_stack/apis/memory_banks/memory_banks.py

```diff
@@ -144,3 +144,15 @@ async def register_memory_bank(
         provider_id: Optional[str] = None,
         provider_memory_bank_id: Optional[str] = None,
     ) -> MemoryBank: ...
+
+    @webmethod(route="/memory_banks/update", method="POST")
+    async def update_memory_bank(
+        self,
+        memory_bank_id: str,
+        params: BankParams,
+        provider_id: Optional[str] = None,
+        provider_memory_bank_id: Optional[str] = None,
+    ) -> MemoryBank: ...
+
+    @webmethod(route="/memory_banks/delete", method="POST")
+    async def delete_memory_bank(self, memory_bank_id: str) -> None: ...
```

Comment:
I think we should call it `unregister_` in keeping with our `register_` naming.

Reply:
done.